src/main/resources/data-repository-openapi.yaml

```diff
@@ -413,18 +414,13 @@ definitions:
     type: object
     required:
       - table
-      - column_list
+      - column
```

Comment:
My gut is this is a mistake. It is small work to build our internal structures to support a list. It will be bigger work later to visit the API, metadata schema, and internal classes to add a list back in.

That said, I'm not going to be in the way of merging this PR.